frontend/src/Buildings/index.jsx

```diff
@@ -60,6 +79,10 @@ function Buildings() {
     // TODO for one building
   }
 
+  function addPress(type) {
```

Comment:
```suggestion
  function addNewBuilding(type) {
```

in a more general case (like a reusable button) `handleClick` or `handlePress` or something would be OK, but this function is specifically for creating a new building, so the name should be obvious about the action it does